demos/link-prediction/random-walks/ctdne-link-prediction.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>Interestingly, the AUC for the temporal and static method are almost the same  over and above being better than the paper's results.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Yeah I tried to figure out for a while whether the way I've set up is cheating in some way but decided to move forward eventually... so let me know if you spot something!